axelrod/player.py

```diff
@@ -98,6 +98,15 @@ def play(self, opponent, noise=0):
             s1, s2 = self._add_noise(noise, s1, s2)
         update_histories(self, opponent, s1, s2)
 
+    def reproduce(self):
```

Comment:
Sure we could change the name... I also thought about "duplicate". I went with reproduce since it was sort of Darwinian (copy but don't inherit acquired state or history). But whatever everyone prefers is fine.